<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MixerForm</class>
 <widget class="QWidget" name="MixerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSlider" name="motor_1_slider">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>22</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1000</number>
   </property>
   <property name="maximum">
    <number>2000</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="motor_2_slider">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>22</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1000</number>
   </property>
   <property name="maximum">
    <number>2000</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="motor_3_slider">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>22</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1000</number>
   </property>
   <property name="maximum">
    <number>2000</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="motor_4_slider">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>22</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1000</number>
   </property>
   <property name="maximum">
    <number>2000</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="motor_all_slider">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>100</y>
     <width>22</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1000</number>
   </property>
   <property name="maximum">
    <number>2000</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="motor_1_value">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1000</string>
   </property>
  </widget>
  <widget class="QLabel" name="motor_2_value">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1000</string>
   </property>
  </widget>
  <widget class="QLabel" name="motor_3_value">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1000</string>
   </property>
  </widget>
  <widget class="QLabel" name="motor_4_value">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>270</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1000</string>
   </property>
  </widget>
  <widget class="QLabel" name="motor_all_value">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>270</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1000</string>
   </property>
  </widget>
  <widget class="QLabel" name="motor_1_value_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Motor 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="motor_1_value_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Motor 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="motor_1_value_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Motor 3</string>
   </property>
  </widget>
  <widget class="QLabel" name="motor_1_value_5">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Motor 4</string>
   </property>
  </widget>
  <widget class="QLabel" name="motor_1_value_6">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>70</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>All</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="enableMotorTestMode">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Motor Text Mode</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
